crud.c

```diff
@@ -0,0 +1,203 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct info
+{
+    int id;
+    char name[50];
+    int age;
+}u;

```

Comment:
Give proper name instead of "u" like "user" or something suitable according to code requirement